docs/usage/logging.md

```diff
@@ -0,0 +1,126 @@
+# Logging stack
+
+### When to use the logging stack?
+Currently Kubernetes uses the underlying container runtime logging which does not persist logs for not existing containers. This makes it difficult to investigate issues occurred in long ago destroyed containers.
+
+
+### Components:
+* A Fluent-bit daemonset which works like a log collector and custom custom Golang plugin which spreads log messages to their Loki instances
+* One Loki Statefulset in the `garden` namespace which contains logs for the seed cluster and one per shoot namespace which contains logs for shoot's controlplane.
+* One Grafana Deployment in `garden` namespace and two Deployments per shoot namespace (one exposed to the end users and one for the operators). Grafana is the UI component used in the logging stack.
+
+### How to access the logs
+The first step is to authenticate in front of the Grafana ingress. The secret with the credentials can be found in garden-<project> namespace under <shoot-name>.monitoring.
```

Comment:
...that feels unfinished. What do I expect to find in the secret? Can I add more than one user? What type of authentication should i expect? Just sample questions you should consider answering.
Do no assume I'm in front of a shell and with a project. Assume i'm researching if it's ok to use Gardener just by reading this document.

Reply:
In this secret you can find the credentials for the first logging in the User Grafana.